src/universal/components/ListFilterSet/ListFilterSet.js

```diff
@@ -3,6 +3,8 @@ import cssModules from 'react-css-modules';
 import styles from './ListFilterSet.scss';
 
 @cssModules(styles)
+// for the decorators
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
the stateless function error doesn't really pertain to decorators. we should still be able to wrap a stateless function just fine. let's let it ride for now, change it in the upcoming CSS to react-look PR.